### PR TITLE
TT-448: add mservctl update command

### PR DESCRIPTION
**Pull request by AaronFeledy**
## Description
While mserv supports updating bundles, the mservctl client does not currently implement this functionality. This PR implements it by adding an `mservctl update` command to the tool.

Usage:
`mservctl update 9c9ecec1-8f98-4c3f-88cd-ca3c27599e6b bundle.zip`

## Related Issue
#15 

## Motivation and Context
Currently, mservctl only allows us to delete and recreate plugin entries each time we make a code update to one of our plugins. Our plugins are shared in multiple APIs and we promote them through dev->stage->prod environments. This creates a lot of work as each API needs to be updated with the new plugin ID in each environment following each new update to the plugin. The ability to update the existing plugin saves us from much of this work.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by mitjaziv** [comment]
@AaronFeledy Thank you for the effort, I will check this and get back to you as soon as possible.

File: mservctl/cmd/update.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+func updateMiddleware(cmd *cobra.Command, args []string) {
+	file, err := os.Open(args[1])
+	if err != nil {
+		log.WithError(err).Error("Couldn't open the bundle file")
+		return
+	}
+	defer file.Close()
+
+	params := mw.NewMwUpdateParams().WithID(args[0]).WithUploadFile(file)
```
Comment (komuw) [inline mservctl/cmd/update.go:33]: Hi @AaronFeledy 
Given your other pull request, https://github.com/TykTechnologies/mserv/pull/26 , where we set the timeout to 2minutes. Would this also require a similar timeout?

Comment (AaronFeledy) [reply mservctl/cmd/update.go]: It would. I have updated it to include a timeout.

**Review by mitjaziv — APPROVED** [review]
@AaronFeledy this looks great, thank you for you contribution and sorry it took so long.

**Pull request merged**